templates/email-signatures.md

```diff
@@ -0,0 +1,32 @@
+# Email Signatures
+
+A few examples to base your signature off of. Choose your own title, it can be whatever you wish.
+
+```
+Richard Hendricks
+
+CEO
+---------------
+Niteo.co
+
+E: rh@niteo.co
+W: www.niteo.co
+```
+
+```
+Erlich Bachmann
+
+Backend Specialist
+------------------
+Niteo.co
+
+E: eb@niteo.co
+W: www.niteo.co
+```
+
+For the minimalist, we would suggest this slimmer signature:
+
+```
+Monica Hall, Senior (con)Sultan
+niteo.co | +386 40 123 456 | skype: monica.hall | @mhall
```

Comment:
I just want to give credit where credit is due: this format was "stolen" from @andraztori. It was love at first sight.